jscomp/core/js_packages_info.ml

```diff
@@ -110,6 +110,11 @@ let module_system_of_string package_name : module_system option =
   | "es6-global" -> Some Es6_global
   | _ -> None
 
+let module_system_to_string = function
```

Comment:
Not sure if this other function could be reused:

https://github.com/melange-re/melange/blob/0aa8e5ab60a583c1227c169211184aab86d2fc5d/mel/bsb_package_specs.ml#L63-L67

Reply:
Unfortunately there's no intersection between these 2 modules in either `melc` or `mel`. It's a good area of improvement.